requirements_dev.txt

```diff
@@ -5,3 +5,6 @@ mongomock>=4.1.2,<5
 pylint>=2.15.5,<3
 pytest>=7.2.0,<8
 python-semantic-release>=7.32.2,<8
+mypy
```

Comment:
Would be good to indicate at least a minimum version (the one you tested with). You could, for example, use the comptability matcher (`~=`) on the minor version (e.g., `mypy~=1.8`) to match all `1.x` versions that are at least `1.8.0` (i.e., equivalent to `mypy>=1.8,==1.*`) or you could just pin the minimum version (`mypy>=1.8.0`).